src/data_morph/shapes/points.py

```diff
@@ -131,6 +132,150 @@ def __init__(self, dataset: Dataset) -> None:
         )
 
 
+class _LemniscateBernoulli(PointCollection):
```

Comment:
Let's merge this with the `Infinity` class (calling it `Infinity`). The docs have a broken link with this because they ignore anything prefixed with `_`, and this class has all the functionality of the `Infinity` class already.